src/ensembl/src/content/app/entity-viewer/gene-view/components/transcripts-filter/TranscriptsFilter.tsx

```diff
@@ -126,11 +124,8 @@ const TranscriptsFilter = (props: Props) => {
 
   return (
     <div className={styles.container}>
-      <div className={styles.filterLabel} onClick={props.toggleFilter}>
-        {props.label}
-        <ChevronUp className={styles.chevron} />
-      </div>
       <div className={filterBoxClassnames}>
+        <div></div>
```

Comment:
we shouldn't add empty elements to our markup

Reply:
thats to create the empty space before the "sort by".  it seems cleaner than adding margin-left. what would you suggest?

Reply:
padding-left on the `filterBox`

Reply:
padding-left is the same as margin-left in terms of behavior on smaller screen/different browser.

Reply:
> padding-left is the same as margin-left in terms of behavior on smaller screen/different browser.

What benefit would an empty first column give for small screens or different browsers?

Reply:
So i might be wrong here but i thought with margin-left or padding-left the spacing will be different on different browsers but when i tried it out, it seems to work fine (specially when resizing the window). Could be older browsers had this issue. So i have now removed the extra div and added padding-left to the first div.